providers/implementations/asymciphers/rsa_enc.c

```diff
@@ -164,13 +164,10 @@ static int rsa_encrypt(void *vprsactx, unsigned char *out, size_t *outlen,
             PROVerr(0, ERR_R_INTERNAL_ERROR);
             return 0;
         }
-        ret =
-            rsa_padding_add_PKCS1_OAEP_mgf1_with_libctx(prsactx->libctx, tbuf,
-                                                        rsasize, in, inlen,
-                                                        prsactx->oaep_label,
-                                                        prsactx->oaep_labellen,
-                                                        prsactx->oaep_md,
-                                                        prsactx->mgf1_md);
+        ret = ossl_rsa_padding_add_PKCS1_OAEP_mgf1_with_libctx(
+                    prsactx->libctx, tbuf, rsasize, in, inlen,
+                    prsactx->oaep_label, prsactx->oaep_labellen,
+                    prsactx->oaep_md, prsactx->mgf1_md);
```

Comment:
I dunno for the moment if we have an actual rule to align this kind of list with the parenthesis, but I find this particular "solution" astoundingly ugly.  It it was me, I'd do this:

``` C
        ret = ossl_rsa_padding_add_PKCS1_OAEP_mgf1_with_libctx
            (prsactx->libctx, tbuf, rsasize, in, inlen,
             prsactx->oaep_label, prsactx->oaep_labellen,
             prsactx->oaep_md, prsactx->mgf1_md);
```

Having to move your eyes waaaaay to the right to see the parenthesis that tells youthat's a function parameter list, that's not the generally considered good readability, as far as I know.

And now, I'm not saying the my variant is *beautiful*, but I do think it's *better*

Reply:
Ughh.....I find that much worse :-(

Reply:
You might want to run that past @DDvO's source code checker for a more technical opinion :wink:

Reply:
As usual, there is all manner of possibilities existent in the code.
I'd prefer your style but double indented so the args start under the function name.

Reply:
I just wonder how the source checker will react, not to mention if it's even possible to have an indent program do that, should we choose to do that exercise again.

Reply:
`check-format.pl` was fine with the original indentation,
but (as expected) is not with the current one in the PR,
while the version suggested by Richard is good and accepted by the tool.

Reply:
I will happily defer to check-format.pl on this one :laughing: